### PR TITLE
feat(contentful): Add sys property check for select

### PR DESCRIPTION
**Pull request by Khaledgarbaya**
TODO:

- [x] Add Integration test

### CODE REVIEW

File: lib/create-contentful-api.js [hunk 1]
```diff
@@ -230,7 +232,15 @@ export default function createContentfulApi ({http, shouldLinksResolve}) {
...
+  /*
+   * sdk relies heavily on sys metadata
+   * so we cannot omit the sys property on sdk level
+   * */
+  function normalizeSelect (query) {
+    if (query.select && (!/sys/i.test(query.select) && !/sys /i.test(query.select))) {
+      query.select += ',sys'
+    }
+  }
   return {
```
Comment (stefanjudis) [inline lib/create-contentful-api.js]: no empty line here?

File: lib/create-contentful-api.js [hunk 2]
```diff
@@ -230,7 +232,15 @@ export default function createContentfulApi ({http, shouldLinksResolve}) {
   function parseEntries (data) {
     return wrapEntryCollection(data, true, false)
   }
-
+  /*
+   * sdk relies heavily on sys metadata
+   * so we cannot omit the sys property on sdk level
+   * */
+  function normalizeSelect (query) {
+    if (query.select && (!/sys/i.test(query.select) && !/sys /i.test(query.select))) {
```
Comment (stefanjudis) [inline lib/create-contentful-api.js]: I don't understand the second regexp...
What's the deal with this space?

Comment (Khaledgarbaya) [inline lib/create-contentful-api.js]: I'll probably remove since adding a space before will be an invalid url

**Comment by stefanjudis** [comment]
LGTM. 🚢

**Pull request merged**